test/maverick/node/node.go

```diff
@@ -1469,25 +1484,15 @@ func createAndStartPrivValidatorSocketClient(
 }
 
 func createAndStartPrivValidatorRPCClient(
-	host string,
-	initialQuorumHash crypto.QuorumHash,
 	defaultQuorumType btcjson.LLMQType,
-	username string,
-	password string,
+	dashCoreRpcClient *dashcore.RpcClient,
 	logger log.Logger,
 ) (types.PrivValidator, error) {
-
-	pvsc, err := privval.NewDashCoreSignerClient(host, username, password, defaultQuorumType)
+	pvsc, err := privval.NewDashCoreSignerClient(dashCoreRpcClient, defaultQuorumType)
 	if err != nil {
 		return nil, fmt.Errorf("failed to start private validator: %w", err)
 	}
 
-	// try to get a pubkey from private validate first time
-	_, err = pvsc.GetPubKey(initialQuorumHash)
```

Comment:
The point of getting the public key verifies that this core client can act as a private validator, this should not be removed.